topics/computational-chemistry/tutorials/htmd-analysis/tutorial.md

```diff
@@ -41,6 +41,36 @@ This tutorial provides an introduction to using high-throughput molecular dynami
 >
 {: .agenda}
 
+
+# Abbreviations
+* **Hsp90** –Heat Shock Protein 90 <br>
```

Comment:
```suggestion
* **Hsp90** - Heat Shock Protein 90
```
Can you make sure you format the same on each line? With 1 space before and after the dash.

Also, the `<br>` tag should not be needed.